lib/taskana-simplehistory-provider/pom.xml

```diff
@@ -166,6 +166,11 @@
 			<artifactId>mybatis</artifactId>
 			<version>3.4.6</version>
 		</dependency>
+		<dependency>
+			<groupId>pro.taskana</groupId>
+			<artifactId>taskana-core</artifactId>
+			<version>1.0.4-SNAPSHOT</version>
```

Comment:
We should not depent on snapshots, right?

Reply:
Yes. I will change it.